pm_icecon/land_spillover.py

```diff
@@ -0,0 +1,125 @@
+"""Routines which implement land spillover corrections to sea ice concentration fields.
+
+land_spillover.py
+"""
+
+import os
+
+import numpy as np
+from scipy.signal import convolve2d
+
+
+def read_adj123_file(
```

Comment:
Consider adding type annotations for the parameters in this function and others in this file.

Reply:
done